### DISCUSSION TITLE
Który CSP jest dobry. Polecenia

### DISCUSSION THREAD

**Discussion by chomamirek** [AzureCommunityPL/group#279] [Ogólna]
Poszukuje kontaktu do Microsoftu albo do sprawdzonego CSP który pomoże w kwestiach wdrożenia chmury w małym start upie. 
Ktoś coś? Jakieś polecenia?

**Comment by mimachniak** [comment]
Ja z Tech Data mam bardzo miłe doświadczenie dla startup

**Comment by szympulka** [comment]
Ja jestem podpięty pod APN Promise. Zdarzyło mi się wiele razy pisać do supportu i nigdy nie miałem problemu. Myślę, że głównym wyznacznikiem będzie cena :D

**Reply by miqm** [reply]
APN Promise jest dystrybutorem a nie CSP. Oni nie mogą Ci sprzedać subskrypcji na Twoje potrzeby, mogą tylko dostarczyć subskrypcję tobie jako CSP abyś sprzedał ją swojemu klientowi.

**Reply by smereczynski** [reply]
Dla startupu polecam TechData.

**Reply by krukowskid** [reply]
Jak coś to śmiało można pisać do APN. Przekierują temat bezpośrednio do jednego z partnerów. Jakby ktoś potrzebował to podziele się mejlem/telefonem do odpowiednich osób. Z wdrożeniem chmury też pomogą 👍
